src/App/theme.ts

```diff
@@ -278,6 +279,17 @@ const theme = createMuiTheme({
         borderRadius: 8
       }
     },
+    MuiSwitch: {
```

Comment:
You checked that the existing switches still look good? 😉

Reply:
All other switches look the same since all of them have `color='primary'`. The one for toggling testnet is the only one that uses `color='secondary'`.